api/src/main/java/org/eclipse/microprofile/config/Config.java

```diff
@@ -90,11 +92,11 @@
  */
 @org.osgi.annotation.versioning.ProviderType
 public interface Config {
-
     /**
-     * Return the resolved property value with the specified type for the
-     * specified property name from the underlying {@linkplain ConfigSource configuration sources}.
+     * Return the resolved property value with the specified type for the specified property name from the underlying
+     * {@linkplain ConfigValueSource configuration sources}.
```

Comment:
I think you need to mention ConfigSource as there might be no ConfigValueSource provided.

Reply:
It really should not matter since they'll be wrapped.

Reply:
I started thinking just reuse ConfigSource and add methods instead, which is much cleaner.

Reply:
Idea was to wrap the old ConfigSource in the new ConfigValueSource, so we wouldn't need to refer to both of them all the time.

Reply:
no. I suggest not to introduce ConfigValueSource.